cloudformation/vpc.yml

```
@@ -0,0 +1,254 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: lecture10-vpc-template
+
+
+
+Parameters:
+  #vpc----------------------------------------------------------------------
+  VpcCidrBlock:
+    Description: Input a VPC IPv4 CidrBlock. # ex 192.168.2.0/16
```

```suggestion
    Description: Input a VPC IPv4 CidrBlock. 
```
サンプルを記載するコメントは不要かと。
defaultで十分表現できていると思います！